src/sentry/incidents/tasks.py

```diff
@@ -218,5 +219,11 @@ def process_pending_incident_snapshots():
                     incident.status == IncidentStatus.CLOSED.value
                     and not IncidentSnapshot.objects.filter(incident=incident).exists()
                 ):
-                    create_incident_snapshot(incident, windowed_stats=True)
+                    project_ids = list(
+                        IncidentProject.objects.filter(incident=incident).values_list(
+                            "project_id", flat=True
+                        )
```

Comment:
You don't really need the ids here, can just use `IncidentProject.objects.filter(incident=incident).exists()`